src/Main/Gear.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from 'common/Icon';
+
+class Gear extends React.PureComponent {
+  static propTypes = {
+    selectedCombatant: PropTypes.object.isRequired,
+  };
+  render() {
+    const toolTipBaseUrl='http://www.wowhead.com/item=';
+    const data = Object.values(this.props.selectedCombatant._gearItemsBySlotId);
+    const getBorderColor = (quality) => { // Not used right now
+      switch (quality) {
+        case 6:
+          return '#e5cc80';
+        case 5:
+          return '#ff8000';
+        case 4:
+          return '#a335ee';
+        case 3:
+          return '#0070dd';
+        case 2:
+          return '#1eff00';
+        case 1:
+          return '#ffffff';
+        default:
+          break;
+      }
+    };
+    const artifactData = data.filter(item => item.quality === 6 && item.gems); // get artifact from list of items to render relics
+    return(
+      <div>
+        <div style={{marginLeft: 'auto', marginRight: 'auto', display: 'block', width: '90%'}}>
+          {data.map(item => {
+            if (item.id === 0) return null;
```

Comment:
When working with an array it would be slightly cleaner to use the `filter` method to filter results. Example:

```jsx
{data
  .filter(item => item.id !== 0)
  .map(item => (
    <div>...rest
```